server/src/main/java/com/paypal/selion/utils/ConfigParser.java

```diff
@@ -48,6 +48,10 @@
      *         represented by the JSON file passed via the JVM argument <b>SeLionConfig</b>
      */
     public static ConfigParser parse() {
+        LOGGER.entering();
+        if (parser == null) {
```

Comment:
@sundaramrajendran Yes, I see the bug now. Thanks!  This said, I think `ConfigParser` needs a few more changes.

1. We should add a constructor with the `configFile` as a parameter.
2. We should not call `readConfigFileContents` in any constructor.
3. We should wait to call `readConfigFileContents` on the first invocation of `parse`